tasks/set_vars.yml

```diff
@@ -4,7 +4,28 @@
 - name: Lookup applications and set the apps var
   set_fact: apps="{{ lookup('pipe', 'python plugins/utils/lookup_apps.py') | from_yaml }}"
 
-- name: Display looked up applications
+# If the playbook is invoqued with the app_filter extra var, we restrict apps
+# to the list passed as an argument, e.g.:
+#
+#     $ ansible-playbook foo.yml -e "app_filter='richie,redirect'"
```

Comment:
You don't think apps_filter is a better name for this vars ?

Reply:
Seems relevant.